docs/guides/1228-go-ci.md

```diff
@@ -0,0 +1,33 @@
+---
+slug: /1228/go-ci
+displayed_sidebar: europa
```

Comment:
This should be `displayed_sidebar: '0.2'`

Reply:
done

Reply:
Actually, it [failed in netlify](https://app.netlify.com/sites/devel-docs-dagger-io/deploys/625803bfaebc310009fddbd5#L80)

Reply:
Oops, forgot to rebase.
I guess the PR with the banner got merged :)

Reply:
Rebased, but still, it doesn't show in the sidebar. What am I missing?

Reply:
It needs to be added to `website/sidebar.js` explicitly, like this: https://github.com/dagger/dagger/blob/629604fa217c026d42ea8cd5a785dad04fe89d10/website/sidebars.js#L42-L58

Reply:
This is the only thing left AFAIK before we can merge. Everything else we can address post-merge. Keep it flowing ⛲️